connectors add hint to skip encoding for rest outbound (#4837)
camunda/camunda-docs: docs/components/connectors/protocol/rest.md

}
```

> ### Encoding:

Reviewer: ```suggestion
### Encoding
```

> In certain scenarios, such as when working with APIs that require pre-encoded URL elements, the REST Connector's default behavior may inadvertently modify encoded segments. To address this, you can set the `skipEncoding` value to `"true"` in the XML, which disables the automatic decoding and re-encoding process, ensuring the URL is sent to the server exactly as provided.

Reviewer: ```suggestion
In certain scenarios, such as when working with APIs that require pre-encoded URL elements, the REST Connector's default behavior may inadvertently modify encoded segments. 

To avoid this, set the `skipEncoding` value to `"true"` in the XML. This disables the automatic decoding and re-encoding process, ensuring the URL is sent to the server exactly as provided.
```
Just a small bit of rewording/spacing 👍
